docs/cardinal/game/configuration.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: 'Configuration'
+description: 'Configuring the Cardinal game shard'
+---
+
+Each service in the World Engine stack reads its configuration values from environment variables. Using the [World CLI](https://github.com/Argus-Labs/world-cli) makes configuring your project even easier via its `world.toml` file, where all World Engine configuration values are in one place.
+
+Below are the key value pairs that can be set in either the `world.toml` file or directly in environment variables.
+
+
+## Cardinal `[cardinal]`
+| Name                         | Description                                                                                                                                                                                                          | Required for Production |
+|------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------|
+| CARDINAL_NAMESPACE           | The name of your cardinal game shard. If you are running multiple game shards in a single World Engine instance, this MUST be unique for each game shard. Must only be alphanumeric characters and hyphens (-).      | YES                     |
```

Comment:
Cardinal should be capitalized.

brand/product names should always be capitalized (i.e. World Engine, Redis, Cardinal, Nakama, etc)

Reply:
fixed [nits](https://github.com/Argus-Labs/world-engine/pull/736/commits/ec1345c83691c5e1ea44e62550f72af71b52e2d1)